patches/@patternfly+pfe-core+4.0.4.patch

```diff
@@ -0,0 +1,46 @@
+diff --git a/node_modules/@patternfly/pfe-core/decorators/observes.js b/node_modules/@patternfly/pfe-core/decorators/observes.js
```

Comment:
out of curiosity where are you using the observes decorator on an object?

Reply:
it's because the function signature of standard decorators is different

see the source code for `lit/decorators/property.js`

Reply:
we could probably revert this file and save it for a later PR